ReviewMonitor/Model/App.swift

```diff
@@ -16,7 +16,7 @@ class App: NSObject {
 
     init(dict: [String: Any]) {
         name = dict["name"] as! String
-        bundleId = dict["bundle_id"] as! String
+        bundleId = dict["bundle_id"] as? String ?? "-"
```

Comment:
Do you know in what case `bundle_id` will be `nil` on iTunes Connect? Isn't it a required field when you create an app there?

Reply:
When an app has no submitted builds, you can set the bundle ID to none. As I'm having now.

Reply:
Got it. Thanks 👍. Please revert the project file changes and then I will merge it.